README.md

```diff
@@ -116,6 +111,8 @@ The [Packer template](src/packer.pkr.hcl) defines a number of variables:
 | release\_tag | The GitHub release tag to use for the tags applied to the created AMI. | `string` | `""` | no |
 | release\_url | The GitHub release URL to use for the tags applied to the created AMI. | `string` | `""` | no |
 | skip\_create\_ami | Indicate if Packer should not create the AMI. | `bool` | `false` | no |
+| winrm\_password | The password used to connect to the instance via WinRM. | `string` | `false` | no |
```

Comment:
Just for documentation we'll present this as a required variable.

```suggestion
| winrm\_password | The password used to connect to the instance via WinRM. | `string` | n/a | yes |
```

Reply:
@mcdonnnj resolved with [2973420](https://github.com/cisagov/windows-server-packer/pull/10/commits/29734201bc39508d11674b968a889785ba6b5cb6)